tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -324,7 +599,67 @@ system_health/test_system_health.py::test_service_checker_with_process_exit:
       - "build_version <= '20201231.44'"
 
 #######################################
-#####           VxLAN             #####
+#####       test_pretest.py       #####
+#######################################
+test_pretest.py::test_stop_pfcwd:
+  skip:
+    reason: "Skip this test on non dualTOR testbeds."
+    conditions:
+      - "'dualtor' not in topo_name"
+
+#######################################
+#####            vlan             #####
+#######################################
+vlan/test_vlan.py::test_vlan_tc7_tagged_qinq_switch_on_outer_tag:
+  skip:
+    reason: "Unsupported platform."
+    conditions:
+      - "asic_type not in ['mellanox', 'barefoot']"
+
+#######################################
+#####             voq             #####
+#######################################
+voq/test_voq_ipfwd.py:
+  skip:
+    reason: "Did not find any asic in the DUTs (linecards) that are connected to T1 VM's."
+    conditions:
+      - "not any('T1' in value['name'] for key, value in minigraph_neighbors.items())"
+
+voq/test_voq_ipfwd.py::test_host_route_table_bgp:
+  skip:
+    reason: "Has no inband interfaces, so must not be connected to fabric"
+    conditions:
+      - "len(BGP_VOQ_CHASSIS_NEIGHBOR) == 0"
+
+voq/test_voq_ipfwd.py::test_host_route_table_inband_addr:
+  skip:
+    reason: "Has no inband interfaces, so must not be connected to fabric."
+    conditions:
+      - "len(VOQ_INBAND_INTERFACE) == 0"
+
+voq/test_voq_nbr.py::TestGratArp::test_gratarp_macchange:
+  skip:
+    reason: "Can't run this test without any IP interfaces on ethernet ports."
+    conditions:
+      - "len(INTERFACE) == 0"
+
+voq/test_voq_nbr.py::test_neighbor_hw_mac_change:
+  skip:
+    reason: "Can't run this test without any IP interfaces on ethernet ports."
+    conditions:
+      - "len(INTERFACE) == 0"
```

Comment:
Are you sure that variables like `BGP_VOQ_CHASSIS_NEIGHBOR`, 'VOQ_INBAND_INTERFACE' and `INTERFACE` are available in the basic facts variables?